circle.yml

```diff
@@ -0,0 +1,82 @@
+version: 2
+workflows:
+  version: 2
+  linux-builds:
+    jobs:
+      - build-x86_64
+      - build-i686
+
+jobs:
+  build-x86_64:
+    docker:
+#      - image: ubuntu:14.04
+      - image: circleci/python:2.7
+        environment:
+          JULIA_CPU_CORES: 4
+          JULIA_TEST_MAXRSS_MB: 800
+          ARCH: x86_64
+#    resource_class: xlarge
+    steps: &steps
+#          apt-get update &&
+#            curl make patch m4 xz-utils git time ccache bar
+      - run: | # install build dependencies
```

Comment:
Rather than commenting a name for the command, you can actually do
```yaml
- run:
    name: install build dependencies
    command: |
      sudo apt-get .......
```
That will display the given name as the collapsible block header when viewing the build on their web UI.